src/go/cmd/content.go

```diff
@@ -323,6 +323,26 @@ func (c *content) filterDeclarations(typ string, decls map[string]Declaration, t
 	return results
 }
 
+// findMethods takes the name of the receiver and looks for Funcs that are methods on that receiver.
+func (c *content) findMethods(s string) map[string]Func {
```

Comment:
This looks a slightly modified `searchForMethods`. If we need both, can one call the other?

Reply:
That was my original intention.  However, `searchForMethods` does a bit more work when ranging over `c.Funcs` so I left them split.  Still, they could be consolidated with a little extra work.

Reply:
Fixed.